.NET/Microsoft.Recognizers.Text.Sequence/SequenceRecognizer.cs

```diff
@@ -11,9 +13,34 @@ private SequenceRecognizer(SequenceOptions options)
         {
             RegisterModel(Culture.English, options.ToString(), new Dictionary<Type, IModel>
             {
+                [typeof(PhoneNumberModel)] = new PhoneNumberModel(new PhoneNumberParser(), new PhoneNumberExtractor())
```

Comment:
If for different languages, it always use the same dictionary, I think we can just reuse it? not need to new so many  objects?